CodeSnippetsReflection.OpenAPI.Test/PowerShellGeneratorTests.cs

```diff
@@ -358,5 +358,191 @@ public async Task GeneratesSnippetForRequestWithWrongQuotesForStringLiteralsInBo
+        }
+
+        [Fact]
+        public async Task GeneratesSnippetForRequestWithApplicationZipContentType()
+        {
+            using var requestPayload = new HttpRequestMessage(HttpMethod.Put, $"{ServiceRootUrl}/me/drive/items/XXXX/content")
+            {
+                Content = new StringContent(
+                    "Zip file content",
+                    Encoding.UTF8,
+                    "application/zip")
+            };
+            var snippetModel = new SnippetModel(requestPayload, ServiceRootUrl, await GetV1SnippetMetadata());
+            var result = _generator.GenerateCodeSnippet(snippetModel);
+            Assert.Contains("Set-MgDriveItemContent", result);
+        }
+
+        [Fact]
+        public async Task GeneratesSnippetForRequestWithImageContentType()
+        {
+            using var requestPayload = new HttpRequestMessage(HttpMethod.Put, $"{ServiceRootUrl}/me/photo/$value")
+            {
+                Content = new StringContent(
+                    "Binary data for the image",
+                    Encoding.UTF8,
+                    "image/jpeg")
+            };
+            var snippetModel = new SnippetModel(requestPayload, ServiceRootUrl, await GetV1SnippetMetadata());
+            var result = _generator.GenerateCodeSnippet(snippetModel);
+            Assert.Contains("Set-MgUserPhotoContent", result);
+            Assert.Contains("-BodyParameter", result);
+        }
+
+        [Fact]
+        public async Task GeneratesBetaSnippetForFunctionsWithoutParams()
+        {
+            using var requestPayload = new HttpRequestMessage(HttpMethod.Get, $"{ServiceRootBetaUrl}/contacts/delta()?$select=displayName%2CjobTitle%2Cmail");
+            var snippetModel = new SnippetModel(requestPayload, ServiceRootBetaUrl, await GetV1SnippetMetadata());
+            var result = _generator.GenerateCodeSnippet(snippetModel);
+            Assert.Contains("Get-MgBetaContactDelta", result);
+        }
+
+        [Fact]
+        public async Task GeneratesBetaSnippetForFunctionsWithSingleParam()
+        {
+            using var requestPayload = new HttpRequestMessage(HttpMethod.Get, $"{ServiceRootBetaUrl}/drives/XXXX/items/XXXX/delta(token='token')");
+            var snippetModel = new SnippetModel(requestPayload, ServiceRootBetaUrl, await GetV1SnippetMetadata());
+            var result = _generator.GenerateCodeSnippet(snippetModel);
+            Assert.Contains("Get-MgBetaDriveItemDelta", result);
+            Assert.Contains("-Token", result);
+        }
+
+        [Fact]
+        public async Task GeneratesBetaSnippetForFunctionsWithMultipleParam()
+        {
+            using var requestPayload = new HttpRequestMessage(HttpMethod.Get, $"{ServiceRootBetaUrl}/communications/callRecords/getPstnBlockedUsersLog(fromDateTime=XXXXXX,toDateTime=XXXXX)");
+            var snippetModel = new SnippetModel(requestPayload, ServiceRootBetaUrl, await GetV1SnippetMetadata());
```

Comment:
You should call `GetBetaSnippetMetadata` instead of `GetV1SnippetMetadata` for beta tests. The same applies to all beta test.
```suggestion
            var snippetModel = new SnippetModel(requestPayload, ServiceRootBetaUrl, await GetBetaSnippetMetadata());
```